graalpy-eval-demo/README.md

```diff
@@ -0,0 +1,57 @@
+# GraalPy Evaluating Demo
+This small application demonstrates how to use GraalPy to evaluate a Python file from a Java application using Maven.
+The application contains a Main class that evaluates an external Python file named _Hello.py_ when run.
+
+The example also shows how to produce a native executable using [GraalVM](https://www.graalvm.org/).
+
+
+## Preparation
+
+Install a JDK 21, ensure that it is on your classpath, and set the value of `JAVA_HOME` accordingly.
+We recommend GraalVM for JDK 21 that can be easily installed using [SDKMAN!](https://sdkman.io/).
+```bash
+sdk install java 21.0.2-graal
+```
```

Comment:
We should probably add also:

For other download options, go to [GraalVM Downloads page](https://www.graalvm.org/downloads/).